frontend_v2/src/app/components/challenge/challengesubmit/challengesubmit.component.ts

```diff
@@ -230,6 +246,21 @@ isSubmissionUsingUrl: any;
     this.authToken = this.globalService.getAuthToken();
   }
 
+  progress() {
```

Comment:
Seems like the progress status is independent of the size of file (or the actual upload time). Please correct me if I am missing something here.

Reply:
yes @lunayach I was just giving the 5 second time for uploading during which i was showing the slider. Now I have changed the time for upload to depends on the size of the file. Please take a look.